tests/migration/migration.go

```diff
@@ -625,18 +627,16 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				By("Checking that the VirtualMachineInstance console has expected output")
 				Expect(console.LoginToAlpine(vmi)).To(Succeed())
 
-				pods, err := virtClient.CoreV1().Pods(vmi.Namespace).List(context.Background(), metav1.ListOptions{
-					LabelSelector: v1.CreatedByLabel + "=" + string(vmi.GetUID()),
-				})
-				Expect(err).ToNot(HaveOccurred(), "Should list pods successfully")
-				Expect(pods.Items).To(HaveLen(1), "There should be only one VMI pod")
+				pod, err := libpod.GetPodByVirtualMachineInstance(vmi, vmi.Namespace)
```

Comment:
This is wrong as `GetPodByVirtualMachineInstance` always returns the new Pod. I would preserve the one Pod assert.

Reply:
I would argue that at this stage there will be only 1 pod or the `GetPodByVirtualMachineInstance` returns an error, but I don't insist